app/AvailablePlugin/EmailAddressWidget/Model/EmailAddressWidgetVerification.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * COmanage Registry Email Address Widget Model
+ *
+ * Portions licensed to the University Corporation for Advanced Internet
+ * Development, Inc. ("UCAID") under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * UCAID licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * @link          http://www.internet2.edu/comanage COmanage Project
+ * @package       registry-plugin
+ * @since         COmanage Registry v4.1.0
+ * @license       Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ */
+
+App::uses("EmailAddress", "Model");
+App::uses("CoMessageTemplate", "Model");
+
+class EmailAddressWidgetVerification extends AppModel {
+  // Cache the record to verify
+  private $_recordToVerify = null;
+
+  // Define class name for cake
+  public $name = "EmailAddressWidgetVerification";
+
+  // Add behaviors
+  public $actsAs = array('Containable');
+
+  // Association rules from this model to other models
+  public $belongsTo = array(
+    "CoEmailAddressWidget" => array(
+      'foreignKey' => 'co_email_address_widget_id'
+    )
+  );
+
+  // Validation rules for table elements
+  public $validate = array(
+    'email' => array(
+      'content' => array(
+        'rule' => array('email'),
+        'required' => false,
+        'allowEmpty' => false,
+        'message' => 'Please enter a valid email address'
```

Comment:
Email can't be both not required and not allowed empty. In this case, `required` should probably be `true`.

Also, don't hardcode english error messages. I know there are some places that still do this, but we shouldn't be adding any more.